src/models/brackets.rs

```diff
@@ -513,8 +509,33 @@ pub struct PlayerInfo {
     pub points: i32,
     /// see [points]
     pub opponent_points: i32,
-    /// seconds
-    pub times: Vec<u32>,
+    results: Vec<PlayerResult>,
+}
+
+impl PlayerInfo {
+    /// total time of all races, with forfeits counting as 3 hours (for use in sorting)
+    fn time_adjusted(&self) -> u32 {
```

Comment:
`f32`s don't sort because of stuff about NaNs or subnormals or whatever. but we have whole number seconds laying around anyway